quiz.js

```diff
@@ -37,7 +51,27 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'stark'
```

Comment:
You were consistent with it here, but just remember to be careful about capitalization and case-sensitivity.